policy_trainers/multi_agent_rllib.py

```diff
@@ -4,23 +4,20 @@
 
 import gymnasium as gym
 import ray
-
 from ray.rllib.algorithms import ppo
-from ray.rllib.utils.test_utils import (
-    add_rllib_example_script_args,
-    run_rllib_example_script_experiment,
-)
+from ray.rllib.utils.test_utils import add_rllib_example_script_args, run_rllib_example_script_experiment
 from ray.tune.logger import pretty_print
 from ray.tune.registry import get_trainable_cls
 
+# Import registers hyperdrive envs
+# from agent0.traiderdaive import FullHyperdriveEnv
```

Comment:
dead comment?

Reply:
yeah this was in the previous file, isort moved this chunk. I'll remove this comment